content/document.js

```diff
@@ -676,6 +676,7 @@ function remove(
   const docs = [slug, ...children.map(({ metadata }) => metadata.slug)];
 
   if (dry) {
+    Redirect.add(locale, [[url, redirect]], { dry });
```

Comment:
What if you want to delete a document without a redirect?

Reply:
Is that not possible to do? And if you do, it's a rare case where you don't need the CLI tool but can use good old `git rm` :)

Reply:
I'm following the steps here: https://github.com/mdn/content#deleting-a-document and it doesn't seem to work anymore or the usage has changed and the docs need to be updated.  It feels a lot safer using the CLI to me anyway and the docs advice it, too.